data/json/items/comestibles/wheat.json

```diff
@@ -12,239 +12,247 @@
     "symbol": "%",
     "quench": -2,
     "healthy": 1,
-    "calories": 52,
+    "calories": 180,
     "description": "Whole-grain wheat cereal.  It's surprisingly good, and allegedly good for your heart.",
     "price": 300,
     "material": "wheat",
-    "volume": 1,
+    "volume": 4,
     "flags": [ "EDIBLE_FROZEN" ],
     "charges": 12,
+    "vitamins": [ [ "vitC", 1 ], [ "iron", 2 ] ],
     "fun": 8
   },
   {
     "type": "COMESTIBLE",
     "id": "wheat",
     "name": "wheat",
     "name_plural": "wheat",
-    "weight": 132,
+    "weight": 203,
     "color": "brown",
     "comestible_type": "FOOD",
     "symbol": ",",
     "quench": -10,
     "healthy": 1,
-    "calories": 174,
+    "calories": 667,
     "description": "Raw wheat, not very tasty.",
     "price": 120,
     "material": "wheat",
-    "volume": 4,
+    "volume": 1,
     "flags": [ "EDIBLE_FROZEN" ],
-    "charges": 4,
```

Comment:
Because you changed the charges of this item, you need to divide the weight and price by the number of charges.

In wheat.json, you change charges on lines (other than this one):
41, 98, 138, 200, 319, 339, 394, 441, 463, 487